src/foam/u2/crunch/UCJReferenceView.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch',
+  name: 'UCJReferenceView',
+  extends: 'foam.u2.View',
+
+  imports: [
+    'crunchController',
+    'userCapabilityJunctionDAO'
+  ],
+
+  requires: [
+    'foam.u2.stack.Stack',
+    'foam.u2.stack.StackView'
+  ],
+
+  css: `
+    ^ .foam-u2-stack-StackView {
+      padding-left: 0px !important;
+    }
+  `,
+
+  properties: [
+    {
+      name: 'localStack',
+      factory: function () {
+        return this.Stack.create();
+      }
+    }
+  ],
+
+  methods: [
+    function initE() {
+      this
+        .addClass(this.myClass())
+        .tag(this.StackView.create({
+          data: this.localStack,
+          showActions: false
+        }));
+      (async () => {
```

Comment:
Why have an async block rather than just making initE async?

Reply:
I thought that wasn't okay because the runtime might decide to run the entire function asynchronously even though it starts with synchronous code. It seems like it doesn't and this works every time, but I can't seem to find any information to say if it's guaranteed or undefined behaviour.